dev/cognitiveservices/data-plane/Language/questionanswering-generate.json

```diff
@@ -404,39 +326,36 @@
           "type": "array",
           "description": "The set of documents to process as part of qna generation job.",
           "items": {
-            "$ref": "#/definitions/QnAGenDocument"
+            "$ref": "#/definitions/Document"
           }
         },
-        "qnaGenerationParameters": {
-          "description": "Parameters that define max number of QnAs to be generated, fetch raw documents or not, etc.",
-          "$ref": "#/definitions/QnAGenParameters"
+        "parameters": {
+          "description": "Parameters that define max number of question answers to be generated, fetch raw contents of documents or not, etc.",
+          "$ref": "#/definitions/GenerationParameters"
         }
       }
     },
-    "QnAGenerationResult": {
+    "GenerationResult": {
       "type": "object",
-      "description": "QnA Generation job with documents and questions generated.",
+      "description": "Result of question answer generation job.",
       "properties": {
         "documents": {
           "type": "array",
           "description": "The set of documents with processed raw contents.",
           "items": {
-            "$ref": "#/definitions/QnAGenDocument"
+            "$ref": "#/definitions/Document"
           }
         },
-        "qnas": {
+        "value": {
           "type": "array",
           "description": "The set of question and answers generated in the qna generation job.",
           "items": {
-            "$ref": "#/definitions/GeneratedQnAs"
+            "$ref": "#/definitions/GeneratedQAs"
           }
-        },
-        "nextLink": {
-          "type": "string"
         }
       }
     },
-    "QnAGenDocument": {
+    "Document": {
```

Comment:
At least for the SDK, this will be too generic. I recommend `GenerationDocument` going with the prevailing naming convention herein.